.github/workflows/build_and_test.yml

```diff
@@ -1349,7 +1349,7 @@ jobs:
       - name: Start Minikube
         uses: medyagh/setup-minikube@v0.0.20
         with:
-          kubernetes-version: "1.34.0"
+          kubernetes-version: "1.35.0"
           # Github Action limit cpu:2, memory: 6947MB, limit to 2U6G for better resource statistic
```

Comment:
BTW, this statement is outdated, now GitHub Actions provides runner with 4-vCPUs, 16 GiB of memory

https://docs.github.com/en/actions/reference/runners/github-hosted-runners#standard-github-hosted-runners-for-public-repositories